graph_theory/helpers.py

```diff
@@ -38,7 +38,7 @@ def print_map_as_bits(map: list[list[int]]) -> None:
 
 def present_map(map: list[list[int]]) -> None:
     """
-    Prints simple map visualization and count of the map islands
+    Print simple map visualization, count the islands on the map
```

Comment:
Maybe: Prints simple map visualization, counts the islands on the map

Reply:
It dosen't make a big difference. I just prefer it this way, but we can do yours. Maybe @tdxa will help us with this dilema ;p